### PR TITLE
Add support for rawurl in Apt

### PR DESCRIPTION
**Pull request by rbuch**

### CODE REVIEW

File: src/lib/Bcfg2/Server/Plugins/Packages/Apt.py [hunk 1]
```diff
@@ -34,8 +35,12 @@ def get_config(self):
 
         for source in self:
             if source.rawurl:
-                self.logger.info("Packages: Skipping rawurl %s" %
-                                 source.rawurl)
+                if source.rawurl[-1] != '/':
+                    source.rawurl = source.rawurl + "/"
+                index = string.rfind(source.rawurl, "/", 0, -1)
```
Comment (AlexanderS) [inline src/lib/Bcfg2/Server/Plugins/Packages/Apt.py]: Maybe you mean:

```
index = source.rawurl.rfind('/', 0, -1)
```

`string` is meant to by the variable holding the string.

Comment (rbuch) [inline src/lib/Bcfg2/Server/Plugins/Packages/Apt.py]: It should work either way:

```
>>> import string
>>> string.rfind("testtest", "s", 0, 8)
6
>>> "testtest".rfind("s", 0, 8)
6
```

I can change it if you think it's better style.

**Comment by rbuch** [comment]
I'm not sure why the Travis CI build is failing now, the 2.7 builds are passing, but the 2.6 builds seem to have malformed `pip` commands.

File: src/lib/Bcfg2/Server/Plugins/Packages/Apt.py (hunk 1 above)
Comment (AlexanderS) [inline src/lib/Bcfg2/Server/Plugins/Packages/Apt.py]: Yes, I would change that. The [python docs](https://docs.python.org/2/library/string.html#deprecated-string-functions) states, that you should consider this functions in the string library as deprecated.

**Comment by AlexanderS** [comment]
Yes seems like travis updated the images and the new pip refuses `--use-mirrors`. #297 would fix this, but I would like to cleanup the commits in #297 before merging.

File: src/lib/Bcfg2/Server/Plugins/Packages/Apt.py (hunk 1 above)
Comment (rbuch) [inline src/lib/Bcfg2/Server/Plugins/Packages/Apt.py]: I've fixed this in the latest commit.

**Comment by AlexanderS** [comment]
You can rebase this now on master to fix the build.

**Comment by solj** [comment]
@AlexanderS Looks like the checks pass now. :+1: for me.

**Pull request merged**